lib/semantics/check-omp-structure.h

```diff
@@ -121,6 +132,7 @@ class OmpStructureChecker : public virtual BaseChecker {
   };
   // back() is the top of the stack
   const OmpContext &GetContext() const { return ompContext_.back(); }
+  const OmpContext &GetPrevContext() const { return ompContext_.rbegin()[1]; }
```

Comment:
There have to be at least 2 elements on the stack for this to be valid, right? Is that something the caller is supposed to ensure? If so, you should add a check that they are keeping their promise.

Same thing with `GetContext()`, I suppose. It may not be called when the stack is empty, right?

Reply:
`CHECK` was added to enforce the empty check.